module/VuFind/src/VuFind/ChannelProvider/Facets.php

```diff
@@ -236,26 +236,27 @@ protected function buildChannel(Results $results, $filter, $title,
             'title' => $title,
             'providerId' => $this->providerId,
         ];
+        $retVal['token'] = $this->getToken($filter, $title);
```

Comment:
I changed this in order to provide a unique identifier for each channel. Passing the tokens allows the interface to ID and group channels based on their unique tokens. We can still rely on the presence or absence of content to choose how we handle channels. [The template was doing that already](https://github.com/vufind-org/vufind/pull/693/files#diff-60cdfca206c9cd4f18238a7383bc5192R14).